src/hotspot/share/compiler/compilerDefinitions.hpp

```diff
@@ -54,7 +54,7 @@ enum MethodCompilation {
 };
 
 // Enumeration to distinguish tiers of compilation
-enum CompLevel {
+enum CompLevel : s1 {
```

Comment:
Hope it won't cause gcc to generate inefficient code manupulating bytes.

Reply:
This enum change might be hard to back-port to jdk11, which still uses an older toolchain, at least for Oracle builds.